### DISCUSSION TITLE
Keep-selection doesn't seem to work any more with -modi

### DISCUSSION THREAD

**Discussion by s0la** [davatorium/rofi#2022] [(Possible) Issues]
Hi,

Sorry for bothering you about the old issue, but I recently noticed the syntax you implemented for my earlier feature request [Keep selected item position after selection](https://github.com/davatorium/rofi/issues/1064#) doesn't seem to work any longer..

I thought it's probably me using it the wrong way, but then I tried the sample test script you provided (it's linked in the feature request), and it seems to stop working there as well.. Could you try the script and let me know if you're seeing the same behavior?

I'm running version 1.7.3.

Thanks!

**Comment by DaveDavenport** [comment]
You are running a very old rofi. 
if you click on the commit in the issue, you see you need 1.7.4 or up
https://github.com/davatorium/rofi/commit/daf3127d3a79602112fca747ad71ca06fb8a6f05

**Reply by s0la** [reply]
Oh, totally missed that, sorry for bother you.. I just recently got a laptop with the Debian preinstalled/preconfigured, and given how old my feature request was, I honestly didn't even think it could be possible it doesn't have such an old version in the official repo yet.. Damn Debian, this is our last rodeo.. Sorry for bothering, again, and thanks for the quick response! As always, appreciate your work! :)
